bin/delete-record.js

```diff
@@ -37,6 +37,13 @@ ovh.request('GET', `/domain/zone/${dom.domain}.${dom.tld}/record`, {
       process.exit(2);
     }
 
+    ovh.request('POST', `/domain/zone/${dom.domain}.${dom.tld}/refresh`, (refreshErr) => {
+      if (refreshErr) {
+        console.error(refreshErr);
+        process.exit(1);
+      }
+    });
+
     process.exit(0);
```

Comment:
Thank you again for contributing!
Unfortunately it doesn't work yet, the async POST call to refresh the zone never happens because `process.exit(0)` is called before. You could simply delete that line, since it's completely redundant. Also, I personally like to use different exit codes, so I know where the process exited. You could simply go up a number and use `process.exit(3)` instead of `process.exit(1)`.

Reply:
Hey, I am completely new to node.js...
I see, the POST call is actually inside the DELETE call. That was *not* what I intended to do.

About the exit code, in 'bin/create_record.js' you used `process.exit(1)` for exiting when a error occurred. I overlooked that you use different exit codes in 'delete_record.js'.

Reply:
We've all been there! ;)

Well, it is actually good the way you did it. You have to wait for the call back from the DELETE call.
The problem is that `ovh.request('POST',...` puts an async operation on the event loop to not block the thread, meaning it continues the flow, hitting `process.exit(0)` and closing the whole program all together

Reply:
I think simply removing `process.exit(0);` will be enough.